pkg/controller/machine/metrics.go

```diff
@@ -50,6 +50,16 @@ func NewMachineControllerMetrics() *MetricsCollection {
 			Name: metricsPrefix + "errors_total",
 			Help: "The total number or unexpected errors the controller encountered",
 		}),
+		Provisioning: prometheus.NewHistogram(prometheus.HistogramOpts{
+			Name:    metricsPrefix + "provisioning_time_seconds",
+			Help:    "Histogram of times spent from creating the machines to ready state in the cluster",
```

Comment:
`.. from creating a Machine to ready state...`

Reply:
Fixed in b0b10a1445065d1d4fa5f08b8e9f77c508d4e738

Reply:
Oh no, github des not recognize commit hashes?! Anyway, should i squash these 2 wording changes into 1 commit?

Reply:
you'll gonna squash everything into a single commit again anyway ^^

Reply:
also, for future, I don't need you to specify the commits fixing a thread - I just look at the open threads when reviewing and compare them with the current changes of the PR